cmd/install/assets/hypershift_operator.go

```diff
@@ -582,6 +583,23 @@ func (o HyperShiftOperatorDeployment) Build() *appsv1.Deployment {
 		util.DeploymentAddTrustBundleVolume(&corev1.LocalObjectReference{Name: o.AdditionalTrustBundle.Name}, deployment)
 	}
 
+	if o.OpenShiftTrustBundle != nil {
+		deployment.Spec.Template.Spec.Containers[0].VolumeMounts = append(deployment.Spec.Template.Spec.Containers[0].VolumeMounts, corev1.VolumeMount{
+			Name:      "ocm-trusted-ca-bundle",
```

Comment:
ocm-trusted-ca-bundle where is this name coming from?

Reply:
It's from me

Reply:
what does ocm means?

Reply:
openshift-config-managed. I spelled that out to remove confusion with openshift cluster manager.